File: test_scripts/test_script.ts
// Typescript Test Script
function add(a: number, b: number): number {
    return a + b;
}

class Greeter {
    greet(name: string): string {
        return `Hello, ${name}!`;
    }
}

console.log("Sum:", add(5, 3));
const greeter = new Greeter();
console.log(greeter.greet("World"));

File: test_scripts/test_script.py
# Python Test Script
def add(a, b):
    return a + b

class Greeter:
    def greet(self, name):
        return f"Hello, {name}!"

if __name__ == "__main__":
    print("Sum:", add(5, 3))
    greeter = Greeter()
    print(greeter.greet("World"))
